test/inductor/test_cudagraph_trees.py

```diff
@@ -269,6 +269,19 @@ def foo(x, y):
                 captured_output[0]
             )
 
+        @torch._inductor.config.patch("triton.cudagraph_skip_dynamic_graphs", True)
```

Comment:
Should you add an addional test that we properly re-record the shape or is that already covered?